platform-sdk/swirlds-unit-tests/core/swirlds-platform-test/src/main/java/com/swirlds/platform/test/chatter/simulator/EventTracker.java

```diff
@@ -123,14 +119,11 @@ public void registerNewEvent(final ChatterEventDescriptor descriptor, final Inst
     /**
      * Each node should call this method each time it receives a new event, including self events.
      *
-     * @param descriptor
-     * 		describes the event that a node just received
-     * @param nodeId
-     * 		the ID of the node that received the event
-     * @param receiveTime
-     * 		the time when the event was received
+     * @param descriptor  describes the event that a node just received
+     * @param nodeId      the ID of the node that received the event
+     * @param receiveTime the time when the event was received
      */
-    public void registerEvent(final ChatterEventDescriptor descriptor, final long nodeId, final Instant receiveTime) {
+    public void registerEvent(final EventDescriptor descriptor, final long nodeId, final Instant receiveTime) {
```

Comment:
Null annotations and checks missing

Reply:
I am not going to add those here, since I only changed the type.